lib/irb/command/copy.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module IRB
+  module Command
+    class Copy < Base
+      category "Workspace"
+      description "Copy command output to clipboard"
+
+      help_message(<<~HELP)
+        Usage: copy (command)
+      HELP
+
+      def execute(arg)
+        value = irb_context.workspace.binding.eval(arg)
+        output = irb_context.inspect_method.inspect_value(value, colorize: false)
+
+        if clipboard_available?
+          copy_to_clipboard(output)
+          puts "Copied to system clipboard"
+        else
+          temp_file = write_to_tempfile(output)
+          puts "Wrote: #{temp_file}"
```

Comment:
I kinda prefer just printing something like `System clipboard not found` for now instead of writing to a tempfile. I am not against the idea but I want to wait until users without clipboard to propose solutions that could work for them the best. WDYT?

Reply:
Aligned, it seems like a sensible default to me but might not apply to everyone.